src/Test.UnitTests.Sarif.Multitool/Test.UnitTests.Sarif.Multitool.csproj

```diff
@@ -13,48 +13,16 @@
     <IsTestProject>true</IsTestProject>
   </PropertyGroup>
 
-  <ItemGroup>
-    <None Remove="TestData\ConvertCommand\SemmleQlSample.csv" />
-    <None Remove="TestData\ExportRuleDocumentationCommand\ExpectedOutputs\Test.md" />
-    <None Remove="TestData\MergeCommand\ExpectedOutputs\NoInputFiles.sarif" />
-    <None Remove="TestData\PageCommand\elfie-arriba - Copy.sarif" />
-    <None Remove="TestData\PageCommand\elfie-arriba.sarif" />
-    <None Remove="TestData\RebaseUriCommand\ExpectedOutputs\RunWithArtifacts.sarif" />
-    <None Remove="TestData\RebaseUriCommand\RunWithArtifacts.sarif" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <EmbeddedResource Include="TestData\ConvertCommand\SemmleQlSample.csv" />
-    <EmbeddedResource Include="TestData\ExportRuleDocumentationCommand\ExpectedOutputs\Test.md" />
-    <EmbeddedResource Include="TestData\MergeCommand\ExpectedOutputs\NoInputFiles.sarif" />
-    <EmbeddedResource Include="TestData\QueryCommand\elfie-arriba.CSCAN0020.sarif" />
-    <EmbeddedResource Include="TestData\PageCommand\elfie-arriba.sarif" />
-    <EmbeddedResource Include="TestData\RebaseUriCommand\ExpectedOutputs\RunWithArtifacts.sarif" />
-    <EmbeddedResource Include="TestData\RebaseUriCommand\Inputs\RunWithArtifacts.sarif" />
-  </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="FluentAssertions" Version="5.10.2" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
-    <PackageReference Include="Moq" Version="4.13.1" />
-    <PackageReference Include="System.Collections.Immutable" Version="1.7.0" />
-    <PackageReference Include="System.Composition" Version="1.4.0" />
-    <PackageReference Include="xunit" Version="2.4.1" />
-    <PackageReference Include="xunit.runner.console" Version="2.4.1">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-    </PackageReference>
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-    </PackageReference>
+    <PackageReference Include="xunit" Version="2.4.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
+    <PackageReference Include="coverlet.collector" Version="1.2.0" />
```

Comment:
>coverlet.collector [](start = 31, length = 18)

why was this introduced?

Reply:
The sequence was:

1. Rename `Test.UnitTests.Sarif.Multitool` to `Test.UnitTests.Sarif.Multitool.Library`. That way I made sure all the test data files and so on got carried over.
2. Create a _new_ `Test.UnitTests.Sarif.Multitool` project to hold the one test that actually uses the `Program` class.

So apparently, newly created .NET Core xUnit Test projects come equipped with `coverlet.collection` (a code coverage library that VS uses when use select **Test**, **Analyze Code Coverage**.

---
In reply to: [490579714](https://github.com/microsoft/sarif-sdk/pull/2075#discussion_r490579714) [](ancestors = 490579714)